src/sentry/monitors/utils.py

```diff
@@ -41,6 +43,91 @@ def valid_duration(duration: Optional[int]) -> bool:
     return True
 
 
+# Returns serializer appropriate group_ids corresponding with check-in trace ids
+def fetch_associated_groups(
+    trace_ids: List[str], organization_id: int, project_id: int, start: datetime, end
+) -> Dict[str, Set[int]]:
```

Comment:
Want to move the comment into the docstring?
```suggestion
def fetch_associated_groups(
    trace_ids: List[str], organization_id: int, project_id: int, start: datetime, end
) -> Dict[str, Set[int]]:
    """
    Returns serializer appropriate group_ids corresponding with check-in trace ids
    """
```